packages/cozy-client/src/store/realtimes.js

```diff
@@ -22,7 +21,7 @@ const normalizeDoc = (couchDBDoc, doctype) => {
 /**
  * DispatchChange
  *
- * @param {CozyClient} client CozyClient instane
+ * @param {object} client CozyClient instane
```

Comment:
Why removing the CozyClient type?

Reply:
I had issues with tests when moving the functions in the a file that is imported and exported in `/store/index`. I found other commit fixing the issue. Here is the explanation.

```
If `CozyClient` is imported into those files, then unit tests fail.

Using `@param {import('../CozyClient').default} client` syntax
doesn't work either as this syntax is not handled by `jsdoc`.

Jsdoc seems to support this syntax in later versions but then
node engine should be updated.

So client will be temporarily typed as `object` until a better solution
is found.
```